packages/components/src/test/instance-filter-builder/Utils.test.ts

```diff
@@ -155,7 +155,7 @@ describe("filterRuleValidator", () => {
     sinon.restore();
   });
 
-  it("returns error message for invalid numeric rule", () => {
+  it(`returns error message for undefined`, () => {
```

Comment:
It seems to me that the changed test actually tests a different test case than what the previous test did. I think we should keep the old one and add a new one.

Reply:
Same applies for the changed tests below